### ISSUE TITLE
JOSS review: Improve statement of need

### ISSUE THREAD

**Issue by kbarnhart** [passaH2O/dorado#7]
From @dbuscombe-usgs's JOSS review [link to comment](https://github.com/openjournals/joss-reviews/issues/2585#issuecomment-692365791)

openjournals/joss-reviews/issues/2585

> The statement of need should lead with a definition of the term ‘particle tracking’. It is clear to me that this is shorthand for ‘tracking of water – including particles – in a Lagrangian framework’, but it could be clearer to the reader.
I also recommend adding the theory section in the documentation to the paper (https://passah2o.github.io/dorado/background/index.html). It is well written and answers many outstanding questions I have when left with when reading the paper on its own.

**Comment by elbeejay** [comment]
@gassmoeller left a comment on the statement as well ([link to original comment](https://github.com/openjournals/joss-reviews/issues/2585#issuecomment-699174900)):

> You provide a clear statement of need, but I would prefer if you could specify the target audience a bit more clearly. It seems you are aiming at small- to medium-scale particle models and rapid development and use rather than high-performance and massively parallel models, is that correct? If so, please state.

**Comment by elbeejay** [comment]
Hello, we didn't want to automatically close the PR with the merge of #15 but we do believe we've addressed these concerns, and hopefully have made the paper stronger in the process. A new rendered version of the paper is available [here](https://github.com/openjournals/joss-reviews/issues/2585#issuecomment-708569220). 

We've made the following changes:

1. To improve the statement of need, we try to make it more clear that "particle tracking" is analogous to the tracking of water parcels, and we state that the gap we are aiming to fill is specifically for doing this in a Lagrangian framework. To clarify the use-case and scale of problem, we state that this package is meant for "meant for rapid application in conjunction with any landscape-scale 2D hydrodynamic model"

2. We have added some of the theory behind the weighted random walk scheme into the paper "Background" section.
